src/box/iproto.cc

```diff
@@ -1781,6 +1845,17 @@ net_finish_rollback_on_disconnect(struct cmsg *m)
 		iproto_connection_try_to_start_destroy(con);
 }
 
+/** Cancel all inprogress requests of the connection. */
+static void
+tx_process_cancel(struct cmsg *m)
```

Comment:
Nit: "cancel inprogress" may be a better name.

Reply:
Done.

```diff
--- a/src/box/iproto.cc
+++ b/src/box/iproto.cc
@@ -1844,7 +1844,7 @@ net_finish_rollback_on_disconnect(struct cmsg *m)

 /** Cancel all inprogress requests of the connection. */
 static void
-tx_process_cancel(struct cmsg *m)
+tx_process_cancel_inprogress(struct cmsg *m)
 {
        struct iproto_connection *con =
                container_of(m, struct iproto_connection, cancel_msg);
@@ -3490,7 +3490,7 @@ iproto_do_cfg_f(struct cbus_call_msg *m)
        case IPROTO_CFG_DROP_CONNECTIONS: {
                struct iproto_connection *con;
                static const struct cmsg_hop cancel_route[1] =
-                                               {{ tx_process_cancel, NULL }};
+                               {{ tx_process_cancel_inprogress, NULL }};
                rlist_foreach_entry(con, &iproto_thread->connections,
                                    in_connections) {
                        if (!con->is_in_replication &&
```